src/iidm/converter/xml/VoltageLevelXml.cpp

```diff
@@ -147,6 +187,15 @@ void VoltageLevelXml::writeBusBreakerTopology(const VoltageLevel& voltageLevel,
     context.getWriter().writeEndElement();
 }
 
+void VoltageLevelXml::writeCalculatedBus(const Bus& bus, const std::set<unsigned long>& nodes, NetworkXmlWriterContext& context) {
+    context.getWriter().writeStartElement(IIDM_PREFIX, BUS);
+    context.getWriter().writeAttribute(V, bus.getV());
+    context.getWriter().writeAttribute(ANGLE, bus.getAngle());
+    const auto& mapper = [](const unsigned long& i) { return std::to_string(i); };
```

Comment:
unsigned long is OK: it's not necessary to pass  `i` (rename to node) by const reference